mscore/startupWizard.cpp

```diff
@@ -156,6 +156,24 @@ StartupWizardPage3::StartupWizardPage3(QWidget* parent)
       setLayout(layout);
       }
 
+StartupWizardPage4::StartupWizardPage4(QWidget* parent)
+      : QWizardPage(parent)
+      {
+      setTitle(tr("Tours"));
+      QLabel* label = new QLabel(tr("Do you want to see helping tours?"), this);
```

Comment:
I suppose we need to settle on terminology and be consistent, at least for user-visible strings.  The words "help", "tour", "tip", and "tutorial" all seem possibly relevant here.  Probably there are other terms that could be used too.  I don't really have a preference myself, but "helping tours" seems awkward.

Reply:
I think MuseScore isn't the only application to do this sort of thing, let's be consistent with established terminology.

Reply:
Many of the examples I am seeing will not ask the users if they want to see the tours. Rather, they will show a welcome tour and give the user the option to decline all tours then.